cluster/gce/manifests/cloud-controller-manager.manifest

```diff
@@ -0,0 +1,113 @@
+{
+"apiVersion": "v1",
+"kind": "Pod",
+"metadata": {
+  "name":"cloud-controller-manager",
+  "namespace": "kube-system",
+  "labels": {
+    "tier": "control-plane",
+    "component": "cloud-controller-manager"
+  }
+},
+"spec":{
+"securityContext": {
+  {{runAsUser}}
+  {{runAsGroup}}
+  {{supplementalGroups}}
+  "seccompProfile": {
+      "type": "RuntimeDefault"
+  }
+},
+"priorityClass": "system-node-critical",
+"hostNetwork": true,
+"containers":[
+    {
+    "name": "cloud-controller-manager",
+    "image": "gcr.io/k8s-staging-cloud-provider-gcp/cloud-controller-manager:v1.25.2-alpha_ae91c1fc0c443c464a4c878ffa2a4544483c6d1f",
```

Comment:
when does this get updated?

Reply:
See https://github.com/kubernetes/kubernetes/pull/113271#discussion_r1012354100, there is a TODO to:
* Use a tag based on an actual release
* Ensure this version is the latest known good version of CCM